ebos/eclcpgridvanguard.hh

```diff
@@ -206,8 +207,23 @@ public:
 #endif
 
         cartesianIndexMapper_.reset(new CartesianIndexMapper(*grid_));
-
         this->updateGridView_();
+        if (mpiSize > 1) {
+            std::vector<int> cartIndices;
+            cartIndices.reserve(grid_->numCells());
+#if DUNE_VERSION_NEWER(DUNE_GRID, 2,6)
+            ElementMapper locMapper(this->gridView(), Dune::mcmgElementLayout());
+#else
+            ElementMapper locMapper(this->gridView());
+#endif
```

Comment:
I'm probably missing something, but I don't see direct usage of `locMapper` anywhere in this scope.  The only apparent mapping is "local - to - Cartesian" which goes through the `cartesianIndexMapper_`.

Reply:
indeed, copy-paste issue.

Reply:
Nice catch. line 222 should use`it, otherwise this will only work for `CpGrid`.
```
cartIndices.push_back(cartesianIndexMapper_->cartesianIndex(locMapper.index(*locElementIt):
```

Reply:
i'll add it, while ignoring the fact that we are in eclcpgridvanguard.hh :)